symplyphysics/laws/electricity/circuits/waveguides/maximum_electric_field_intensity_of_main_wave_in_rectangular_waveguide.py

```diff
@@ -1,55 +1,88 @@
+"""
+Maximum electric field strength of main wave in rectangular waveguide
+=====================================================================
+
+A rectangular waveguide is a rectangular metal waveguide capable of supporting waves propagating along it.
+The main wave is a transverse electric wave with the first index equal to :math:`1` and the second index equal to :math:`0`.
+
+**Conditions:**
+
+#. The wave propagating in the waveguide must be the main wave.
+#. The waveguide must be rectangular.
+
+..
+    TODO: find link
+"""
+
 from sympy import Eq, solve, pi, sqrt
-from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
-    validate_output, dimensionless)
+from symplyphysics import (
+    units,
+    Quantity,
+    validate_input,
+    validate_output,
+    symbols,
+    clone_as_symbol,
+)
+
+maximum_electric_field_strength = symbols.electric_field_strength
+"""
+Maximum :symbols:`electric_field_strength` in the waveguide.
+"""
 
-## Description
-## A rectangular waveguide is a rectangular metal waveguide capable of supporting waves propagating along it.
-## The main wave is a transverse electric wave with the first index equal to 1 and the second index equal to 0.
-## The first index shows how many half-wave lengths fit horizontally across the cross section. The second index
-## shows how many half-wave lengths fit vertically across the cross section.
+relative_permittivity = symbols.relative_permittivity
+"""
+:symbols:`relative_permittivity` of the insulator.
+"""
 
-## Law is: E = 2 * 120 * pi * a * H / (L * sqrt(er)), where
-## E - maximum electric field intensity in waveguide,
-## a - width of the waveguide cross section,
-## H - magnetic field intensity in waveguide,
-## L - wavelength,
-## er - relative permittivity of the material filling the waveguide.
+width = clone_as_symbol(symbols.length, display_symbol="a", display_latex="a")
+"""
+Horizontal dimension of the waveguide. See :symbols:`length`.
 
-# Conditions:
-# - the wave propagating in the waveguide must be the main wave;
-# - the waveguide must be rectangular.
+..
+    TODO: clarify horizontal, is it related to the plane of oscillation of the waves?
```

Comment:
Something like this: https://www.electronics-notes.com/images/waveguide-rectangular-te-modes-01.svg
You can try to clarify the description.